calico-cloud.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Calico Cloud
+description: Learn about Calico Cloud.
+canonical_url: '/calico-cloud/index'
+---
```

Comment:
Also some pages might be linking to the old page you removed, so you might want to redirect them here.
```suggestion
redirect_from: '/calico-enterprise'
---
```